src/Sentry/Platforms/Android/LogCatAttachmentEventProcessor.cs

```diff
@@ -81,7 +81,7 @@ public SentryEvent Process(SentryEvent @event, SentryHint hint)
             output.Seek(0, SeekOrigin.Begin);
             var bytes = output.ToArray();
 
-            hint.Attachments.Add(new SentryAttachment(AttachmentType.Default, new ByteAttachmentContent(bytes), "logcat.log", "text/logcat"));
+            hint.Attachments.Add(new SentryAttachment(AttachmentType.Default, new ByteAttachmentContent(bytes), "logcat.log", "text/plain"));
```

Comment:
Is this single line the fix for the preview issue?
Why branch off of another feature?

Reply:
@Bruno-garcia
>Is this single line the fix for the preview issue?

Yes, that's just this line

>Why branch off of another feature?

Initially I wanted to make a pr just for this line. 
But I was also planning on writing some device tests so the CI can confirm the changes, and without PR #3694 the logcat files are not attached, so the CI would have automatically failed.

That's why we branch this off #3694.

Reply:
@bricefriha I think the other PR addresses a specific situation in which logcat logs aren't sent. More generally, as long as we're not dealing with SEGSEGV exceptions, we should be getting logcat reports come through though, so it should be possible to test this fix without mixing the two PRs right?

Reply:
@jamescrosswell You're absolutely right; I don't know why, after working on #3461, in my head I thought the logcat files weren't sent at all. 
Let's reset the branch before the first commit of #3694 and add a commit of the `mime-type` change